tests/test_csc.py

```diff
@@ -58,54 +37,65 @@ def basic_make_csc(self, initial_state, config_dir, simulation_mode):
-            await harness.remote.cmd_setChillerTemperature.set_start(
+            await self.remote.cmd_enable.set_start(timeout=20)
+            await self.remote.cmd_setChillerTemperature.set_start(
                 temperature=target_temp, timeout=STD_TIMEOUT
             )
-            await asyncio.sleep(5)
-            self.assertEqual(harness.csc.chillerModel.setTemp, target_temp)
+            await asyncio.sleep(6)
+            self.assertEqual(self.csc.chillerModel.setTemp, target_temp)
 
     async def testPowerOn(self):
-        async with Harness(initial_state=salobj.State.STANDBY) as harness:
-            harness.csc.kiloarcModel.warmupPeriod = 1
-            await harness.remote.cmd_start.set_start(settingsToApply=None, timeout=20)
-            harness.csc.chillerModel.component.response_dict = {
+        async with self.make_csc(
+            initial_state=salobj.State.STANDBY,
+            config_dir=TEST_CONFIG_DIR,
+            simulation_mode=1,
+        ):
+
+            await self.remote.cmd_start.set_start(settingsToApply=None, timeout=20)
+            self.csc.chillerModel.component.response_dict = {
                 b".0103rSetTemp26\r": b"#01030rSetTemp+019040\r",
                 b".0115sStatus_17c\r": b"#01150sStatus_1A1\r",
                 b".0101WatchDog01\r": b"#01010WatchDog2101EA\r",
                 b".0120rWarnLv1ee\r": b"#01200rWarnLv10800DB\r",
                 b".0104rSupplyT46\r": b"#01040rSupplyT+018867\r",
             }
-            await harness.remote.cmd_enable.set_start(timeout=STD_TIMEOUT)
-            await harness.remote.cmd_startCooling.set_start(timeout=STD_TIMEOUT)
+            await self.remote.cmd_enable.set_start(timeout=STD_TIMEOUT)
+            self.csc.kiloarcModel.warmupPeriod = 1
+            await self.remote.cmd_startCooling.set_start(timeout=STD_TIMEOUT)
             await asyncio.sleep(3)  # Sleep while we wait for chiller to start chilling.
-            await harness.remote.cmd_powerLightOn.set_start(timeout=STD_TIMEOUT)
+            await self.remote.cmd_powerLightOn.set_start(timeout=STD_TIMEOUT)
             await asyncio.sleep(0.3)  # Wait for light to start its 1200w startup burst.
             await asyncio.sleep(4)
-            self.assertEqual(harness.csc.kiloarcModel.component.bulbState, 800)
+            self.assertEqual(self.csc.kiloarcModel.component.bulbState, 800)
 
         await asyncio.sleep(10)
 
     async def testPowerOff(self):
-        async with Harness(initial_state=salobj.State.STANDBY) as harness:
-            await harness.remote.cmd_start.set_start(settingsToApply=None, timeout=20)
-            harness.csc.chillerModel.component.response_dict = {
+        async with self.make_csc(
+            initial_state=salobj.State.STANDBY,
+            config_dir=TEST_CONFIG_DIR,
+            simulation_mode=1,
+        ):
+            await self.remote.cmd_start.set_start(settingsToApply=None, timeout=20)
```

Comment:
Note that you _can_ specify `initial_state="salobj.State.DISABLED"` and drop this cmd_start. This works for any initial state. Still...what you have works.